cluster/saltbase/salt/kube-dns/kubedns-rc.yaml.in

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
```

Comment:
I will move this file to cluster/addons/dns and delete what is there once the PR is stable in prod